Code Relay/Code Relay/DetailsView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DetailsView.swift
+//  Code Relay
+//
+//  Created by Sebastian Bolling on 2024-10-08.
+//
+
+import SwiftUI
+
+struct DetailsViewTests: View {
```

Comment:
Curious what the `Tests` suffix is for? 👀

Reply:
This is Xcode 16 autocompleation "help". Will fix